src/functionsToValidate.js

```diff
@@ -0,0 +1,84 @@
+import { keyTypes } from './constants.js';
```

Comment:
I feel like this file should be inside the `utils` directory. And its name could improve a bit, to something like `validatorFunctions` or `parametersValidators`, its current name confuses me a bit, like these functions need validation, we can discuss it if needed

Reply:
I agree with you @mpollingerQualabs. Since this is an old version of the repo, this change will be done on the [this PR](https://github.com/montevideo-tech/cmcd-validator/pull/52). You are going to find the function `validatorFunctions.js` in the folder `src/keyValueValidator`